proposals/stage-0/SECURITY_REPORTING/README.md

```diff
@@ -0,0 +1,60 @@
+# My New Proposal
+>  Stage 0
+
+## Champion
+
+Marcin Hoppe (@marcin_hoppe)
+
+## Description
+
+Currently OpenJS Foundation projects do not have consistently documented standards for responsible reporting and disclosure of security vulnerabilities. This proposal aligns OpenJS Foundation projects with industry best practices in this space.
+
+See also:
+- [Security reporting in OpenJS Foundation projects](https://gist.github.com/MarcinHoppe/b13a870770522c31a8386ada48b2e40f)
+- [Guidelines for responsible reporting and disclosure of security vulnerabilities](https://github.com/nodejs/package-maintenance/blob/master/docs/drafts/security-guidelines.md)
+
+## Required Resources
+
+- Generic security policy template that projects can easily adopt
+- Optionally: vulnerability disclosure platform (VDP)
+
+## Who would be responsible?
+
+The responsibility for adoption of security reporting standards would be jointly owned by the projects and the CPC.
+
+## How would success be measured?
+
+- 100% of Impact, Growth, and At-Large projects have:
+    - Security policy on GitHub
+    - Vulnerability reporting mechanism (email or VDP)
+- 100% projects graduating from Incubation have a security policy on GitHub
+
+## Why this proposal is important
+
+Open source projects that can responsibly accept security reports and respond with a coordinated vulnerability disclosure have a higher chance of being treated as trustworthy by adopters.
+
+## Unresolved Question
+
+- Do we leave it to projects to coordinate disclosure?
+    - Some projects are already prepared, have infrastructure and processes in place. Others don't.
+- Do we need to offer additional resources to projects?
+    - Vulnerability disclosure platform (e.g. HackerOne)?
+    - Staff triage team?
+- Do we need to have a coordinating body?
+    - Security working group at the Foundation level?
+- How to coordinate with similar industry initiatives?
+    - Open Source Security Coalition (https://securitylab.github.com/)
+    - Node.js Ecosystem Security WG (https://github.com/nodejs/security-wg)
+- Should we offer bounties?
+    - What about corporate sponsors that want to do it for selected programs (e.g. Google / AMP case)?
```

Comment:
I think that can complicate things initially. Don't think a report for Electron is the same as a report for webhint. How do we decide the $$ for each project? And then there's the question about budget for this, etc.
Maybe we could send a survey to know how many projects have received a vulnerability report in the last year, how many have bounties, etc. to have a better idea?

Reply:
@molant I agree. Perhaps this could be a follow up proposal when we have the basics of security reporting covered?

Reply:
I think the first step should focus on getting projects to what you have in `How would success be measured?`.  The actual implementation would remain the responsibility of the project.

A follow on might be seeing if we can get free accounts from H1 for all member projects . 

If we have enough interest I do think we should spin up a group at the Foundation level. Please let me know if you'd be interested in participating in a group like that.

Reply:
@mhdawson I agree and clarified that bug bounties remain out of scope of this proposal.